baracoda/orm/base.py

```diff
@@ -0,0 +1,3 @@
+from sqlalchemy.ext.declarative import declarative_base
```

Comment:
Is this SQLAlchemy thing?

Reply:
Yes, flask_alchemy provides the same thing but in a different way as db.Model. But having our own base class was more useful for referring it into alembic config in file to autogenerate migrations:

```baracoda/alembic/env.py```